app/controllers/records_controller.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+class RecordsController < ApplicationController
+  include Records::Filters
+
+  before_action :authorize,
+                :not_guest,
+                :not_author,
+                :not_owner,
+                :not_manager
+
+  respond_to :html
+
+  # GET /records
+  def index
+    @records       = records.order(created_at: :desc).page params[:page]
+    @class_records = @records.first.class
```

Comment:
```suggestion
    @records_class = @records.first.class
```
Perdón, no lo noté antes.

Reply:
genial